packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/src/Providers/ClientProvider.cs

```diff
@@ -164,6 +171,24 @@ protected override FieldProvider[] BuildFields()
                 }
             }
 
+            fields.AddRange(BuildAdditionalClientFields());
```

Comment:
We are calling this method twice, once in the lazy field assignment for `_additionalClientFields` and another here. We can just update this to be `feilds.AddRange(_additionalClientFields.Value)` since we are caching the results of the  BuildAdditionalClientFields method.

Reply:
was this resolved ?

Reply:
yes sorry! i forgot to push the last commit